app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/services/GlossaryService.scala

```diff
@@ -60,34 +59,33 @@ class GlossaryService @Inject()(messagesApi: MessagesApi, langs: Langs, language
       .toSet
   }
 
-  private def getMessagesPairs(lang: Lang): List[(String, List[(String, String)])] = {
+  private def getAllMessagesPairsUnsorted(lang: Lang): GlossaryContent = {
```

Comment:
Name changed to be more explicit.  No point in sorting at this stage, as we will always sort at fetch time.